libraries/cmake/source/mbedtls/CMakeLists.txt

```diff
@@ -0,0 +1,113 @@
+    "${library_root}/library/des.c"
+    "${library_root}/library/dhm.c"
+    "${library_root}/library/ecdh.c"
+    "${library_root}/library/ecdsa.c"
+    "${library_root}/library/ecjpake.c"
+  "${library_root}/library/ecp.c"
+  "${library_root}/library/ecp_curves.c"
+  "${library_root}/library/entropy.c"
+  "${library_root}/library/entropy_poll.c"
+  "${library_root}/library/error.c"
+  "${library_root}/library/gcm.c"
+  "${library_root}/library/havege.c"
+  "${library_root}/library/hkdf.c"
+  "${library_root}/library/hmac_drbg.c"
+  "${library_root}/library/md.c"
+  "${library_root}/library/md2.c"
+  "${library_root}/library/md4.c"
+  "${library_root}/library/md5.c"
+  "${library_root}/library/memory_buffer_alloc.c"
+  "${library_root}/library/nist_kw.c"
+  "${library_root}/library/oid.c"
+  "${library_root}/library/padlock.c"
+  "${library_root}/library/pem.c"
+  "${library_root}/library/pk.c"
+  "${library_root}/library/pk_wrap.c"
+  "${library_root}/library/pkcs12.c"
+  "${library_root}/library/pkcs5.c"
+  "${library_root}/library/pkparse.c"
+  "${library_root}/library/pkwrite.c"
+  "${library_root}/library/platform.c"
+  "${library_root}/library/platform_util.c"
+  "${library_root}/library/poly1305.c"
+  "${library_root}/library/psa_crypto.c"
+  "${library_root}/library/psa_crypto_driver_wrappers.c"
+  "${library_root}/library/psa_crypto_se.c"
+  "${library_root}/library/psa_crypto_slot_management.c"
+  "${library_root}/library/psa_crypto_storage.c"
+  "${library_root}/library/psa_its_file.c"
+  "${library_root}/library/ripemd160.c"
+  "${library_root}/library/rsa.c"
+  "${library_root}/library/rsa_internal.c"
+  "${library_root}/library/sha1.c"
+  "${library_root}/library/sha256.c"
+  "${library_root}/library/sha512.c"
+  "${library_root}/library/threading.c"
+  "${library_root}/library/timing.c"
+  "${library_root}/library/version.c"
+  "${library_root}/library/version_features.c"
+  "${library_root}/library/xtea.c"
+  "${library_root}/library/certs.c"
+    "${library_root}/library/pkcs11.c"
+    "${library_root}/library/x509.c"
+    "${library_root}/library/x509_create.c"
+    "${library_root}/library/x509_crl.c"
+    "${library_root}/library/x509_crt.c"
+    "${library_root}/library/x509_csr.c"
+    "${library_root}/library/x509write_crt.c"
+    "${library_root}/library/x509write_csr.c"
+    "${library_root}/library/debug.c"
+    "${library_root}/library/net_sockets.c"
+  "${library_root}/library/ssl_cache.c"
+  "${library_root}/library/ssl_ciphersuites.c"
+  "${library_root}/library/ssl_cli.c"
+  "${library_root}/library/ssl_cookie.c"
+  "${library_root}/library/ssl_msg.c"
+  "${library_root}/library/ssl_srv.c"
+  "${library_root}/library/ssl_ticket.c"
+  "${library_root}/library/ssl_tls.c"
+  "${library_root}/library/ssl_tls13_keys.c"
+  )
+target_include_directories(thirdparty_mbedtls PRIVATE
+  "${library_root}/include"
+  )
+
+add_definitions(-DMBEDTLS_MD2_C -DMBEDTLS_MD4_C -DMBEDTLS_PEM_PARSE_C
```

Comment:
Use `target_compile_definition` instead, and lose the `-D` prefix.

Reply:
fixed